lib/public/components/common/form/inputs/TimeRangeInputComponent.js

```diff
@@ -0,0 +1,56 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+import { DateTimeInputComponent } from '../../../common/form/inputs/DateTimeInputComponent.js';
+
+/**
+ * @typedef TimeRangeInputComponentOptions
+ * @property {boolean} [seconds=false] states if the input has granularity up to seconds (if not, granularity is minutes)
+ * @property {number} min minimal timestamp accepted by `from` input
+ * @property {number} max maximal timestamp accepted by `to` input
```

Comment:
Its for both inputs, not only from and to

Reply:
applied